openpmd-validator/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "openpmd-validator" %}
+{% set version = "1.1.0.4" %}
+{% set build = 0 %}
+{% set sha256 = "a15153c95350e713866ff44a307548f575bd4a7833b082d65fe2d5d0551d48f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/openPMD/openPMD-validator/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+    - setuptools
+  run:
+    - h5py
+    - {{ pin_compatible('numpy') }}
+    - python
+    - python-dateutil
+
+test:
+  imports:
+    - opmd_viewer
+
+about:
+  home: https://www.openPMD.org
+  license: ISC
+  license_family: ISC
```

According to https://en.wikipedia.org/wiki/ISC_license
```suggestion
  license_family: MIT
```